### DISCUSSION TITLE
Is it possible to use cell2location with non-spatial data?

### DISCUSSION THREAD

**Discussion by sam-israel** [BayraktarLab/cell2location#155] [General] [answered]
Is it possible to use cell2location for deconvolution of non-spatial data?
I am interested in estimating the cell-type specific expression of each gene (starting from bulk RNA-Seq).

Besides knowing whether it is possible, I am interested to know if you would recommend it.

**Comment by vitkl** [comment, marked as answer]
Hi @sam-israel 

It is possible to use cell2location for non-spatial data, however, the distributional assumptions of our data are tailored to UMI-based detection RNA protocols. So you would need to compare cell2location and other methods designed for bulk RNA-seq on your data. We have not benchmarked cell2location on bulk RNA-seq.

**Reply by sam-israel** [reply]
Thank you.
To pursue the same topic further : 

Using Cell2location, it is possible to estimate cell-type specific expression of every gene in the data.

(I) Would this functionality also be possible for bulk RNA-Seq ?
(II) Would it be possible & recommended  to use those estimates to perform cell-type aware differential expression analysis between two tissues - to see the differential expression **by cell type**?
